perun-openapi/openapi.yml

```diff
@@ -2374,6 +2374,16 @@ components:
         items:
           type: string
 
+    listOfAllowedGroupStatuses:
+      name: "allowedGroupStatuses[]"
+      description: "list of allowed group statuses List<String>"
+      in: query
+      required: false
+      schema:
+        type: array
+        items:
+          type: string
+
```

Comment:
You can also add the `enum` and specify the allowed values.

Reply:
`Enum` added.